code-challenges.test.js

```diff
@@ -64,4 +124,25 @@ const accountTransactions2 = [250, -89, 100, -96]
 const accountTransactions3 = []
 // Expected output: []
 
+describe("accumulatingSum", () => {
+  it("returns an array of the accumulating sum", () => {
+    expect(accumulatingSum(accountTransactions1)).toEqual([100, 83, 60, 51])
+    expect(accumulatingSum(accountTransactions2)).toEqual([250, 161, 261, 165])
+    expect(accumulatingSum(accountTransactions3)).toEqual([])
+  })
+})
+// FAIL  week-3-assessment-jricks86/code-challenges.test.js
+// ● accumulatingSum › returns an array of the accumulating sum
+
+//   ReferenceError: accumulatingSum is not defined
+
 // b) Create the function that makes the test pass.
+// input: array
+// output: array of the accumulating sum.
+// usee the reduce method on the array to add up all the elements in the array and return their cumulative sum.
+// take two arguments a and b, and return their sum. The initial value for the accumulation is 0, which is the second argument passed to reduce.
+
+
+accumulatingSum = (array) => {
+  return array.reduce((a, b) => a + b, 0)
+}
```

Comment:
I see the idea, you're just missing some additional necessary logic. You are getting the accumulating sum, but doing nothing with it. You want to be placing each accumulated sum as a value in an array!